File: web3rpc/rpc-specs/code-samples/python/personal/sendValueTransfer.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

tx = {
    "from": "0x1d4e05bb72677cb8fa576149c945b57d13f855e4",
    "to": "0xafa3f8684e54059998bc3a7b0d2b0da075154d66",
    "value": "0x1230000000"
}
passphrase = "helloWorld"

w3 = Web3(Web3.HTTPProvider(host))
personal_response = w3.personal.send_value_transfer(tx, passphrase)

print(personal_response)

File: web3rpc/rpc-specs/code-samples/javascript/personal/sendValueTransfer.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");

    const tx = {
        "from": "0x1d4e05bb72677cb8fa576149c945b57d13f855e4",
        "to": "0xafa3f8684e54059998bc3a7b0d2b0da075154d66",
        "value": "0x1230000000"
    };
    const passphrase = "helloWorld";

    provider.personal.sendValueTransfer(tx, passphrase, {}, (err, data, response) => {
        console.log(data);
    });
}
)()